model/src/test/java/com/javahelp/model/token/TokenTest.java

```diff
@@ -0,0 +1,49 @@
+package com.javahelp.model.token;
+
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.User;
+import org.joda.time.*;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Unit tests for {@link Token}
+ */
+public class TokenTest {
+
+    // Setting up the user and token
+    ClientUserInfo info = new ClientUserInfo("user1@example.com", "New York",
+            "111","David", "Smith");
+    User user1 = new User("123", info, "user1");
+    Token token = new Token("This is Test token", user1);
+
+    @Test(timeout = 50)
+    public void getToken(){
+        token.setToken("[!jkl123");
+        String expected_token = "[!jkl123";
```

Comment:
make sure to keep Java naming conventions in mind, use camelCase instead of snake_case